Casks/l/lookaway.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+cask "lookaway" do
+  version "1.4.1"
+  sha256 "0f211bad3302453f28bbde874ede70e5c048d1ff09e8d53eadbb977b40a9e0b3"
+
+  url "https://github.com/mysticalbits/lookaway-releases/releases/download/#{version}/LookAway.dmg",
+      verified: "github.com/mysticalbits/lookaway-releases/"
+  name "LookAway"
+  desc "Reduce dry eyes and digital strain"
```

Comment:
Maybe something like the first option?  I was working on the second option below but doesn't really tell what the application is for.  Thoughts? @kushsolitary?
```suggestion
  desc "Break time reminder app"
```
```suggestion
  desc "Reduce digital eye strain"
```

Reply:
The first options sounds good to me @krehel